apps/cross-chain/app/components/polkadotEco.tsx

```diff
@@ -0,0 +1,85 @@
+"use client";
+
+import { SubmittableExtrinsic } from "@polkadot/api/promise/types";
+import { Signer } from "@polkadot/api/types";
+import { getChainConnector, Thea } from "@polkadex/thea";
+
+const SOURCE_CHAIN = "AssetHub";
+const DESTINATION_CHAIN = "Polkadex";
+
+const fromAddress = "5GLFKUxSXTf8MDDKM1vqEFb5TuV1q642qpQT964mrmjeKz4w";
+const toAddress = "5GLFKUxSXTf8MDDKM1vqEFb5TuV1q642qpQT964mrmjeKz4w";
+const amount = 0.01;
+
+export const PolkadotEco = () => {
+  const { getAllChains } = new Thea();
+
+  const queryBalances = async () => {
+    const assetHubChain = getAllChains().find((c) => c.name === SOURCE_CHAIN);
+    if (!assetHubChain) throw new Error(`${SOURCE_CHAIN} chain not found..`);
+    const assetHub = getChainConnector(assetHubChain.genesis);
+    const assets = assetHub.getSupportedAssets();
+    const balances = await assetHub.getBalances(fromAddress, assets);
+    console.log(balances);
+  };
+  const xcmTransfer = async () => {
+    const { web3Enable, web3FromAddress } = await import(
+      "@polkadot/extension-dapp"
+    );
+
+    await web3Enable("polkadex-thea");
+    const injector = await web3FromAddress(fromAddress);
+
+    if (!injector) throw new Error("Injector not found..");
+
+    const assetHubChain = getAllChains().find((c) => c.name === SOURCE_CHAIN);
+    if (!assetHubChain) throw new Error(`${SOURCE_CHAIN} chain not found..`);
+    const assetHub = getChainConnector(assetHubChain.genesis);
+
+    const usdtAsset = assetHub.getSupportedAssets()[1];
+    const destChain = assetHub.getDestinationChains(usdtAsset)[0];
+
+    console.log("Doing transfer...");
+    const transferConfig = await assetHub.getTransferConfig(
+      destChain,
+      usdtAsset,
+      fromAddress,
+      toAddress
+    );
+
+    const ext = await transferConfig.transfer<SubmittableExtrinsic>(amount);
+    console.log(ext);
+
+    const res = await ext.signAndSend(fromAddress, {
+      signer: injector.signer as Signer,
+    });
+    console.log(res);
```

Comment:
Do we need this log?

Reply:
Actually, i just put it to see the output, as this is just a test-app. This is not going to be deployed anywhere. If i remove it, i have to put it again when i add new chain or asset to see the outputs.